ui-admin/package.json

```diff
@@ -44,6 +46,10 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "prestart": "npm run generate",
+    "prebuild": "npm run generate",
+    "generate": "npm run gen-expression-parser",
+    "gen-expression-parser": "(cd ../core/src/main/antlr/bio/terra/pearl/core/antlr; antlr -Dlanguage=TypeScript -o ../../../../../../../../../ui-admin/src/generated/search-exp-parser CohortRule.g4 -visitor -no-listener)",
```

Comment:
Ugly command, but for some reason I _have_ to execute it from the directory of the CohortRule or antlr just freaks out. So, I have to have the obscenely long `../../` trail.